README.md

```diff
@@ -53,6 +53,15 @@ end
 Everypolitician::Index.new.all_legislatures.each do |legislature|
   puts "#{legislature.name} in #{legislature.country.name} has #{legislature.person_count} member(s)"
 end
+
+# Fetch Everypolitician::Popolo::LegislativePeriod for the latest term
+# Note that this may involve a **substantial download**
+assembly = Everypolitician.country(slug: 'Albania').legislature(slug: 'Assembly')
```

Comment:
This isn't really the way we want people loading countries. We want everything to go through EveryPolitician::Index. And we might as well show off the `lower_house` approach here too.